cmd/draupnir-create-instance

```diff
@@ -96,31 +96,63 @@ listen_addresses = 'localhost'
 EOF
 chmod 640 "${INSTANCE_PATH}/postgresql.auto.conf"
 
+# Provision a pg_ident.conf, and ensure that it can't be edited
+# The system username must match the CN provisioned in the client cert above.
+cat > "${INSTANCE_PATH}/pg_ident.conf" <<EOF
+# MAPNAME       SYSTEM-USERNAME                               PG-USERNAME
+draupnir        "Draupnir instance ${INSTANCE_ID} client"     draupnir
+EOF
+
+chown root:postgres "${INSTANCE_PATH}/pg_ident.conf"
```

Comment:
`chown root:draupnir-instance "${INSTANCE_PATH}/pg_ident.conf"`

Reply:
Doh! Obviously I didn't handle that merge conflict as well as I thought I had. Thanks for picking this up